js/ui/scheduler/shaders/ui.scheduler.current_time_shader.vertical.js

```diff
@@ -1,98 +1,110 @@
         this._$bottomShader = $('<div>').addClass(DATE_TIME_SHADER_BOTTOM_CLASS);
-        this._$bottomShader.width(width - this._workspace.getCellWidth()) && this._$bottomShader.height(height);
+        this._$bottomShader.width(width - this._workSpace.getCellWidth()) && this._$bottomShader.height(height);
 
-        this._$bottomShader.css('left', this._getShaderOffset(i, width - this._workspace.getCellWidth()));
+        this._$bottomShader.css('left', this._getShaderOffset(i, width - this._workSpace.getCellWidth()));
 
         $shader.append(this._$bottomShader);
-    },
+    }
 
-    _renderAllDayShader: function(shaderWidth, i) {
-        if(this._workspace.option('showAllDayPanel')) {
+    _renderAllDayShader(shaderWidth, i) {
+        if(this._workSpace.option('showAllDayPanel')) {
             this._$allDayIndicator = $('<div>').addClass(DATE_TIME_SHADER_ALL_DAY_CLASS);
-            this._$allDayIndicator.height(this._workspace.getAllDayHeight());
+            this._$allDayIndicator.height(this._workSpace.getAllDayHeight());
             this._$allDayIndicator.width(shaderWidth);
             this._$allDayIndicator.css('left', this._getShaderOffset(i, shaderWidth));
 
-            this._workspace._$allDayPanel.prepend(this._$allDayIndicator);
+            this._workSpace._$allDayPanel.prepend(this._$allDayIndicator);
         }
-    },
+    }
 
-    _getShaderOffset: function(i, width) {
-        return this._workspace.getGroupedStrategy().getShaderOffset(i, width);
-    },
+    _getShaderOffset(i, width) {
+        return this._workSpace.getGroupedStrategy().getShaderOffset(i, width);
+    }
 
-    _getShaderTopOffset: function(i) {
-        return this._workspace.getGroupedStrategy().getShaderTopOffset(i);
-    },
+    _getShaderTopOffset(i) {
+        return this._workSpace.getGroupedStrategy().getShaderTopOffset(i);
+    }
 
-    _getShaderHeight: function(i, width) {
-        return this._workspace.getGroupedStrategy().getShaderHeight();
-    },
+    _getShaderHeight(i, width) {
+        return this._workSpace.getGroupedStrategy().getShaderHeight();
+    }
 
-    _getShaderMaxHeight: function(i, width) {
-        return this._workspace.getGroupedStrategy().getShaderMaxHeight();
-    },
+    _getShaderMaxHeight(i, width) {
+        return this._workSpace.getGroupedStrategy().getShaderMaxHeight();
+    }
 
-    _getShaderWidth: function(i) {
-        return this._workspace.getGroupedStrategy().getShaderWidth(i);
-    },
+    _getShaderWidth(i) {
+        return this._workSpace.getGroupedStrategy().getShaderWidth(i);
+    }
 
-    clean: function() {
-        this.callBase();
+    clean() {
+        super.clean();
 
-        this._workspace && this._workspace._$allDayPanel && this._workspace._$allDayPanel.find('.' + DATE_TIME_SHADER_ALL_DAY_CLASS).remove();
+        this._workSpace && this._workSpace._$allDayPanel && this._workSpace._$allDayPanel.find('.' + DATE_TIME_SHADER_ALL_DAY_CLASS).remove();
```

Comment:
`this._workSpace?._$allDayPanel?.find('.' + DATE_TIME_SHADER_ALL_DAY_CLASS).remove();` would be much shorter

Reply:
Maybe use interpolation here: `'.' + DATE_TIME_SHADER_ALL_DAY_CLASS` ?